docs/workflow/testing/libraries/testing-apple.md

```diff
@@ -16,14 +16,17 @@ Unlike XHarness-Android, XHarness for iOS is able to boot simulators by its own.
 - The following shows how to run tests for a specific library on a simulator
 ```
 cd src/libraries/System.Numerics.Vectors/tests
-~runtime/dotnet.sh build /t:Test /p:TargetOS=iOS /p:TargetArchitecture=x64
+./dotnet.sh build /t:Test /p:TargetOS=iOS /p:TargetArchitecture=x64
```

Comment:
this needs to be reworked to be like below where you don't `cd` into the dir first